src/Themes/VuetifyTheme/Recipes/GridSystem.recipe.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "Grid System Shortcodes",
+    "displayName": "Grid System Shortcodes",
+    "description": "Adds shortcode capabilities for a grid system.",
+    "author": "StatCan Digital Innovation",
+    "website": "digital.statcan.gc.ca",
+    "version": "1.0.0",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [],
+    "steps": [
+        {
+          "name": "ShortcodeTemplates",
+          "ShortcodeTemplates": {
+            "vcol": {
+              "Content": "{% if Args.Text != nil %}\r\n<v-col>{{Args.Text | sanitize | raw }}</v-col>\r\n{% else %}\r\n<v-col>{{ Content | sanitize | raw }}</v-col>\r\n{% endif %}",
```

Comment:
For Args.Text, I think instead of outputting the arg in the vrow, we want to use it. So if we want to determine the size of the col inside the row by giving it a number (for example [vcol 6] for a half width column), then we want to put that arg in a <v-col> cols prop (<v-col cols='{{ Args[0]'>).